dali/test/python/test_video_pipeline.py

```diff
@@ -628,3 +628,24 @@ def video_decoder_pipeline():
         b = b.as_cpu().as_array()
         absdiff = np.abs(a.astype(int) - b.astype(int))
         assert np.mean(absdiff) < 2, f"error is {np.mean(absdiff)}"
+
+
+def test_2gb_sequence():
```

Comment:
how costly is to run this? Should we mark it as "slow"?

Reply:
CI shows `[success] 4.29% test_video_pipeline.test_2gb_sequence: 2.0841s`
Do you think it is much?

Reply:
I think we can stay. But those slower tests often become a problem in sanitizer builds where everything runs much slower. Let's leave it an see how it goes